examples/app/crossfilter/crossfilter_app.py

```diff
@@ -9,6 +9,6 @@
 @object_page("crossfilter")
 def make_object():
     autompg['cyl'] = autompg['cyl'].astype(str)
-    autompg['origin'] = autompg['cyl'].astype(str)
```

Comment:
I believe this was just a copy/paste error. This made origin have the same data as cyl column.